pkg/tests/tfcheck/exec.go

```diff
@@ -18,22 +18,24 @@ import (
 	"bytes"
 	"os"
 	"os/exec"
-	"strings"
 
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tests/pulcheck"
+	"github.com/stretchr/testify/require"
 )
 
 func execCmd(t pulcheck.T, wdir string, environ []string, program string, args ...string) (*exec.Cmd, error) {
-	t.Logf("%s %s", program, strings.Join(args, " "))
 	cmd := exec.Command(program, args...)
+	require.NoError(t, cmd.Err)
 	var stdout, stderr bytes.Buffer
-	cmd.Dir = wdir
-	cmd.Env = os.Environ()
-	cmd.Env = append(cmd.Env, environ...)
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
+	cmd.Dir = wdir
+	cmd.Env = append(os.Environ(), environ...)
+	t.Logf("%s", cmd.String())
 	err := cmd.Run()
-	t.Logf("error from `%s %s`\n\nStdout:\n%s\n\nStderr:\n%s\n\n",
-		program, strings.Join(args, " "), stdout.String(), stderr.String())
+	if err != nil {
```

Comment:
Are there cases where we could be interested in log output even if there was no error? In that case it would be great to have a flag/env variable to control the behavior here (`Verbose`?)

Reply:
I can't think of any case where we would be interested in stdout if we did not error, since that is used later in the test pipeline. Stderr is never used, so I added a log of that if it's non-empty: https://github.com/pulumi/pulumi-terraform-bridge/pull/2565/commits/3f56c3f0fe108ef9b1af7437a188de2b11495d70.